ui/src/app/pages/appAnalysis/apps-list.tsx

```diff
@@ -9,40 +10,108 @@ import { Button } from 'app/components/buttons';
 import { FadeBox } from 'app/components/containers';
 import { FlexColumn, FlexRow } from 'app/components/flex';
 import { ListPageHeader } from 'app/components/headers';
+import {
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalTitle,
+} from 'app/components/modals';
 import colors from 'app/styles/colors';
-import { withCurrentWorkspace } from 'app/utils';
-import { AnalyticsTracker } from 'app/utils/analytics';
+import { reactStyles, withCurrentWorkspace } from 'app/utils';
+import { APP_LIST } from 'app/utils/constants';
+import { WorkspacePermissionsUtil } from 'app/utils/workspace-permissions';
+
+const styles = reactStyles({
+  fadeBox: {
```

Comment:
Do we have somewhere where this styling could be shared across multiple pages?

Reply:
Its a good idea however i just checked the styling for fadebox are different for different page. We could move some inside the FadeBox component?

Reply:
That's fair. If it is different among all instances, I think keeping the styling separate makes sense for now.